### DISCUSSION TITLE
docker lxc

### DISCUSSION THREAD

**Discussion by titof2375** [tteck/Proxmox#937] [Help Needed]
Hello, I have an lxc docker that I created with your method, the problem I have is that the usb keys cannot be seen. How to do?

**Comment by tteck** [comment]
> 🛈 If the LXC is created Privileged, the script will automatically set up USB passthrough.

Create the LXC privileged

**Reply by titof2375** [reply]
Yes

**Comment by titof2375** [comment]
how do you see if the lxc has a bluetooth key?

**Reply by tteck** [reply]
What does `lsusb` show?
